modular_skyrat/modules/cortical_borer/code/cortical_borer_abilities.dm

```diff
@@ -922,33 +911,33 @@
 
 /datum/action/cooldown/willing_host/Trigger(trigger_flags)
 	if(!IsAvailable())
-		to_chat(owner, span_warning("This action is still on cooldown!"))
+		owner.balloon_alert(owner, "action on cooldown")
 		return
 	if(!iscorticalborer(owner))
 		to_chat(owner, span_warning("You must be a cortical borer to use this action!"))
 		return
 	var/mob/living/simple_animal/cortical_borer/cortical_owner = owner
 	if(!cortical_owner.inside_human())
-		to_chat(cortical_owner, span_warning("You require a host to upgrade!"))
+		owner.balloon_alert(owner, "host required")
 		return
 	if(cortical_owner.host_sugar())
-		to_chat(owner, span_warning("Sugar inhibits your abilities to function!"))
+		owner.balloon_alert(owner, "cannot function with sugar in host")
 		return
 	if(cortical_owner.chemical_storage < 300)
-		to_chat(cortical_owner, span_warning("You require at least 300 chemical units before you can ask your host!"))
+		owner.balloon_alert(owner, "300 chemical points required")
 		return
 	cortical_owner.chemical_storage -= 300
 	for(var/ckey_check in GLOB.willing_hosts)
 		if(ckey_check == cortical_owner.human_host.ckey)
-			to_chat(cortical_owner, span_warning("This host is already willing, try another host!"))
+			owner.balloon_alert(owner, "host already willing")
 			return
-	to_chat(cortical_owner, span_notice("The host is being asked..."))
+	owner.balloon_alert(owner, "asking host...")
 	var/host_choice = tgui_input_list(cortical_owner.human_host,"Do you accept to be a willing host?", "Willing Host Request", list("Yes", "No"))
 	if(host_choice != "Yes")
-		to_chat(cortical_owner, span_warning("The host was not willing!"))
+		owner.balloon_alert(owner, "host not willing!")
 		StartCooldown()
 		return
-	to_chat(cortical_owner, span_notice("The host was willing!"))
+	owner.balloon_alert(owner, "host willing!")
```

Comment:
A very important thing you can miss so easily. Why?

Reply:
Because you in fact get put into the host and if you miss a balloon alert i dunno what to tell you

Reply:
balloon alerts are literally the worst fucking thing for people with certain kinds of disabilities

Reply:
they should be reserved for spammy inconsequential messages. not critical once a round things.